chat/v1/stream.proto

```diff
@@ -13,6 +13,9 @@ import "profile/v1/stream.proto";
 option go_package = "github.com/harmony-development/legato/gen/chat/v1";
 
 // Request type for use in the `StreamEvents` endpoint.
+// By default, this endpoint will subscribe to all events.
+// Any guilds joined in the future will be added to the subscription as well.
+// Use the UnsubscribeFromAll event for more granular control.
```

Comment:
"use the UnsubscribeFromAll event for unsubscribing from all current subscriptions and disable the automatic guild subscriptions."